include/deal.II/differentiation/sd/symengine_optimizer.h

```diff
@@ -1921,6 +1933,48 @@ namespace Differentiation
       SymmetricTensor<rank, dim, ReturnType>
       evaluate(const SymmetricTensor<rank, dim, Expression> &funcs) const;
 
+
+      /**
+       * Returns the result of a value substitution into the optimized
+       * counterpart of @p func. This function fetches that one value
+       * stored in the @p cached_evaluation vector.
```

Comment:
I have to admit that I don't understand what one should pass as that second argument.

Reply:
Ok, I'll cross reference the `evaluate()` function that returns the `const std::vector<ReturnType> &`. A copy of that vector is really the only thing that you'd ever want as an input to this function.

Reply:
I've given a little more detail to the documentation here, as well as at the point to which you indicated in your previous comment.